packages/desktop-client/src/components/reports/reports/CustomReport.jsx

```diff
@@ -19,16 +19,16 @@ import { Text } from '../../common/Text';
 import { View } from '../../common/View';
 import { AppliedFilters } from '../../filters/FiltersMenu';
 import { PrivacyFilter } from '../../PrivacyFilter';
-import { ChooseGraph } from '../ChooseGraph';
+import {ChooseGraph} from '../ChooseGraph';
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏interesting that linter isn't complaining about this
```suggestion
import { ChooseGraph } from '../ChooseGraph';
```